projectiles/CDFCannonMolecularOvercharge01/CDFCannonMolecularOvercharge01_script.lua

```
@@ -1,4 +1,5 @@
-local CMolecularCannonProjectile = import("/lua/cybranprojectiles.lua").CMolecularCannonProjectile
+Same pattern, straightforward merge:
```

Luacheck (1.2.0)

[error] expected '=' near 'pattern'

(E011)